app/controllers/api/v0/base_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V0
+    class BaseController < ApplicationController
+      include Api::OpenidConnect::ProtectedResourceEndpoint
+
+      protected
+
+      def current_user
+        current_token ? current_token.authorization.user : nil
+      end
```

Comment:
I wonder whether we should allow normal Devise authentication here too. Though deciding (and implementing that) is no merge blocker.